browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -4137,17 +4137,19 @@ index 37468432577f64e3d18bc3667ca298f43a3bf503..81e54d04c6142b29241ec5379470ef47
      ResourceRequest request = loader.request();
      if (!!url)
          request.setURL(URL({ }, url));
-@@ -1289,14 +1370,24 @@ Protocol::ErrorStringOr<void> InspectorNetworkAgent::interceptRequestWithRespons
+@@ -1289,14 +1370,25 @@ Protocol::ErrorStringOr<void> InspectorNetworkAgent::interceptRequestWithRespons
      response.setHTTPStatusCode(status);
      response.setHTTPStatusText(statusText);
      HTTPHeaderMap explicitHeaders;
 +    String setCookieValue;
```

Comment:
I seriously doubt this `frame->page()->cookieJar().setCookieFromResponse` code below works. You can't modify cookies from the page process, only network process can do that. Otherwise compromised renderer would be able to poison the httpOnly cookies, etc.

Reply:
As discussed offline this is inline with the rest of current network instrumentation.